streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -62,4 +65,23 @@ public void testGetRestoreConsumerConfigs() throws Exception {
         assertEquals(returnedProps.get(ConsumerConfig.CLIENT_ID_CONFIG), "client-restore-consumer");
         assertNull(returnedProps.get(ConsumerConfig.GROUP_ID_CONFIG));
     }
+
+    @Test
+    public void testSerde() {
```

Comment:
Nitpick: I'd prefer a method name that is more descriptive -- what exactly is being tested w.r.t. serde?  That is properly passes configuration options to the wrapped serializer and deserializer?  That you can roundtrip a piece of data via `thing == deserialize(serialize(thing))`?  It may also help to split the test into two -- one for the config passing and one for the ser/de functionality;  they are related but not the same.

Also, it think this test should perhaps be moved to a different class -- we are testing `Serde` functionality here -- the class `StreamsConfig` is not even involved in this test.  So perhaps `SerdeTest`?